packages/jsx/src/index.ts

```diff
@@ -32,6 +32,8 @@ type DomApis = Pick<
   | 'DocumentFragment'
 >
 
+const isFalsy = (value: unknown): value is JSX.Falsy => typeof value === 'boolean' || value === '' || value == null
```

Comment:
`typeof value === 'boolean'` isn't "falsy", as `true` is not falsy. Please, use `isSkipped`.